README.md

```diff
@@ -44,15 +44,33 @@ To deactivate the virtualenv (after you finished working):
 Install dependencies in virtual environment:
 
     $ pip3 install -r requirements.txt
-    
+
+## Environment Variables
+
+TBD
+
 ## Run Application
 
 Start the server by running:
-  
-    $ python web/app.py
 
-## Unit Test
+    $ export FLASK_ENV=development
+    $ export FLASK_APP=web
+    $ flask run
+
+## Unit Tests
+To run the unit tests use the following commands:
+
+    $ python3 -m venv venv_unit
```

Comment:
I split the tests into unit tests, and integration tests, each with their own virtual environment and dependencies.